### PR TITLE
Replaces current global lists with the GLOB lists

### PR DESCRIPTION
**Pull request by spookydonut**
Done

[why]: # Allows debugging of global lists

### CODE REVIEW

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -1,100 +1,30 @@
...
 var/global/list/active_diseases = list()
 var/global/list/events = list()
 
 //used by binoculars for dropship bombardment
 var/global/list/active_laser_targets = list()
 
 //used by the main overwatch console
-var/global/list/active_orbital_beacons = list()
-var/global/list/active_supply_beacons = list()
+var/global/list/active_orbital_beacons = list()
```
Comment (Rohesie) [inline code/__HELPERS/global_lists.dm]: Any reason for leaving this and a few other global lists out?

**Comment by Rohesie** [comment]
Also conflicts.

File: code/datums/world_topic.dm [hunk 2]
```diff
@@ -102,7 +102,7 @@
 
 /datum/world_topic/server_hop/Run(list/input)
 	var/expected_key = input[keyword]
-	for(var/mob/dead/observer/O in GLOB.player_list)
+	for(var/mob/dead/observer/O in GLOB.GLOB.player_list)
```
Comment (DominikPanic) [inline code/datums/world_topic.dm]: This ain't right, chief.

File: code/datums/world_topic.dm [hunk 3]
```diff
@@ -53,7 +53,7 @@
 	log = FALSE
 
 /datum/world_topic/playing/Run(list/input)
-	return GLOB.player_list.len
+	return GLOB.GLOB.player_list.len
```
Comment (DominikPanic) [inline code/datums/world_topic.dm]: Same here.

**Pull request merged**